packages/vue/src/utils/FormUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { Reference } from "@fkui/logic";
+import { ComponentValidityEvent, FormErrorList } from "../types";
+
+/**
+ * @public
+ */
+export type ComponentReferences =
+    | Record<string, ComponentValidityEvent>
+    | Record<string, Reference<FormErrorList>>;
+
+/**
+ * @public
+ */
+export type ComponentValueTypes =
+    | ComponentValidityEvent
+    | Reference<FormErrorList>;
+
+/**
+ * Clean inputs which doesn't exist in the DOM
+ *
+ * @public
+ */
+export function cleanUpElements(vm: {
```

Comment:
När och hur använder jag denna? Vem sätter `vm.components`?

Reply:
När ska **konsumenten** anropa denna? Jag är fortfarande inte riktigt säker på användningen. Jag förstår internt av komponenten.

Kan den användas av något annat än `FValidationGroup`? Tänker på att den är otroligt specifik att komponenten du skickar in som första argument måste ha en viss struktur så som `FValidationGroup` råkar ha.

Jag tycker vi sätter denna och `ComponentReferences` som privat kanske tom flytta filen in i `FValidationGroup` katalogen tills någon skriker att den visst behövs.

Motiveringen är att det är brytande, vi ska göra en brytande release. Om vi gör det nu och eventuellt backar på det sen så är det bara en buggrättning.

Om vi inte tar bort den borde vi dokumentera den.

Har en ganska stark åsikt om detta.